src/main/java/com/github/tomakehurst/wiremock/store/files/FileSourceBlobStore.java

```diff
@@ -38,7 +41,15 @@ public FileSourceBlobStore(FileSource fileSource) {
 
   @Override
   public Optional<InputStream> getStream(String key) {
-    return Optional.of(fileSource.getBinaryFileNamed(key).getStream());
+    try {
+      return Optional.of(fileSource.getBinaryFileNamed(key).getStream());
+    } catch (Exception exception) {
+      if (!(exception instanceof FileNotFoundException)) {
+        notifier()
+            .info("Error when working with FileSource:\n" + Json.write(exception.getMessage()));
+      }
+      return Optional.empty();
```

Comment:
I think we should still throw the exception if it's not `FileNotFoundException`. Only return empty when it's exactly that exception.

We can probably do this with two catch blocks rather than `instanceof`.

The notifier call should be at `error` if it's not a not-found error.